hack/debug.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
a little confused about the use of this script

Reply:
the script starts a kube-gateway process locally for debugging

Reply:
should we add illustration or comment in readme or somewhere else?

Reply:
ok

Reply:
I will revert debug script commit and push a new PR later